selectors/emojis.ts

```diff
@@ -51,10 +51,15 @@ export function isCustomEmojiEnabled(state: GlobalState) {
     return config && config.EnableCustomEmoji === 'true';
 }
 
-export const getOneClickReactionEmojis = (state: GlobalState) => {
-    const recentEmojis = getRecentEmojis(state).slice(-3);
-    const emojiMap = getEmojiMap(state);
+export const getOneClickReactionEmojis = createSelector(
+    'getOneClickReactionEmojis',
+    getEmojiMap,
+    getRecentEmojis,
+    (emojiMap, recentEmojis) => {
+        if (recentEmojis.length === 0) {
+            return [];
+        }
 
-    const emojis = recentEmojis.map((recentEmoji) => emojiMap.get(recentEmoji)).filter(Boolean).reverse();
-    return emojis;
-};
+        return recentEmojis.map((recentEmoji) => emojiMap.get(recentEmoji)).filter(Boolean).reverse();
```

Comment:
This is slightly changing the behavior, my expectation to keep the behavior consistent with the previous existing code, would be to set here `recentEmojis.slice(-3)`

Reply:
Oops, fixed it. added slice back